product-input-fields-for-woocommerce.php

```diff
@@ -55,6 +55,18 @@ function get_wc_pif_option( $option, $default = false ) {
 	}
 }
 
+require_once 'includes/component/plugin-tracking/class-tyche-plugin-tracking.php';
```

Comment:
@PratapDungrani I mentioned that this should be outside of the `is_admin()` code block and not outside of the plugin class altogether. Please add it back to the `includes()` function, probably at the start of the function [here](https://github.com/TycheSoftwares/product-input-fields-for-woocommerce/blob/ff6bb0673e4768436e28086e9dcb9c7be3a68b90/product-input-fields-for-woocommerce.php#L151)

Reply:
I tried that first but it still did not work and hence I placed it outside class.

Reply:
I have added the code where you have suggested and it worked for me as well. Thanks